tests/python/test_xgboost_integration.py

```diff
@@ -77,3 +77,28 @@ def test_xgb_iris(self):
       assert predictor.pred_transform == 'max_index'
       assert predictor.global_bias == 0.5
       assert predictor.sigmoid_alpha == 1.0
+
+  def test_logistic(self):
+    np.random.seed(0)
+    kRows = 16
```

Comment:
Is there a similar test for the exponential families? You could parametrize and reuse this with just a different randint limit for count:poisson probably.

Reply:
Adding it.